src/devices_models/devices/common/range_constraint.jl

```diff
@@ -49,7 +49,7 @@ function device_range(psi_container::PSIContainer,
             end
             expression_lb = JuMP.AffExpr(0.0, variable[name, t] => 1.0)
             for val in range_data.additional_terms_lb[ix]
-                JuMP.add_to_expression!(expression_lb, get_variable(psi_container, val)[name, t])
+                JuMP.add_to_expression!(expression_lb, get_variable(psi_container, val)[name, t], -1.0)
```

Comment:
Why did this change to -1? the sign should be assigned in the creation of range_data.

Reply:
For downward reserves, the reserve provision needs to be subtracted from the device output and constrained to be above Pmin. However, the reserve provision value is still positive, albeit downward.

Reply:
Makes sense.